styleSheet.css

```diff
@@ -29,22 +29,81 @@ ul {
   float: left;
 }
 
-.navBar__list>a{
+.navBar__list > a{
   color: white;
   display: block;
   padding: 14px 16px;
   text-align: center;
   text-decoration: none;
 }
 
-.navBar__list>a:hover{
+.navBar__list > a:hover{
   background-color: #111;
 }
 
-.navBar__list--active>a{
+.navBar__list--active > a{
   color: #4CAF50;
 }
+.navBar__list__closeBtn{
+  color: #818181;
+  display: none;
+  padding: 5px 5px 0px 0px;
+  text-align: right;
+  text-decoration: none;
+}
+
+#bodyResume {
+  position: relative;
+  top :5%;
+}
+.navBar__btn{
+  background-color: #111;
+  border: none;
+  color: #fff;
+  display: none;
+  font-size: calc(1em + 3.5vw);
+}
+
+.navBar__sidebarBtn{
+  height: 6.795%;
+  position: fixed;
+  width: 12.08%;
+  left: 0;
+  top: 0;
+  opacity: 0.7;
+}
 
-#bodyResume{
+@media only screen and (max-width: 768px) {
+  .navBar__sidebarBtn,
+  .navBar__list__closeBtn,
+  .navBar__btn{
+    display: block;
+  }
+  #navBar{
+    height: 100%;
+    left: 0;
+    overflow-x: hidden;
+    position: fixed;
+    top: 0;
+    transition: 0.5s;
+    width: 50%;
+    z-index: 1;
+  }
+  .navBar__list {
+    width: 100%;
+  }
+  .navBar__list > a{
+    display: block;
+    font-size: calc(1em + 2vw);
```

Comment:
are you having this fluid font size just for the nav? I think you can declare the font-size fluid for all the html

Reply:
it didn't cross my mind, will do.

Reply:
it is just an idea, we need to check how it works in small devices

Reply:
I check I didn't found any weird behavior, but anyway I will be modifying it depending on the section or/and platform